gdk4/src/texture.rs

```diff
@@ -31,3 +31,107 @@ impl<O: IsA<Texture>> TextureExtManual for O {
         }
     }
 }
+
+// TODO Figure out how to implement an async variant for for_pixbuf.
+impl Texture {
+    #[cfg(any(feature = "v4_6", feature = "dox"))]
```

Comment:
All of those new functions could have a:
1 - Very simple code example or a test case if possible
2 - a bit of docs
3 - point to the upstream issue and mention that we should drop those manual implementations once those are fixed upstream